packages/netdog/generate-network-config.service

```diff
@@ -1,9 +1,12 @@
 [Unit]
```

Comment:
nit: the commit message only explains what was done but not why. There is another commit in the series where you used a list and in each item, you explained why you did what you did. You should follow the same convention (or even better, favor a narrative rather than a bullet list)